resources/fixtures/Clay_Paky/Clay-Paky-Alpha-Spot-575-HPE.qxf

```diff
@@ -93,28 +93,28 @@
   <Capability Min="48" Max="55" Preset="GoboMacro" Res1="ClayPaky/gobo00020.svg">Gobo 6</Capability>
   <Capability Min="56" Max="63" Preset="GoboMacro" Res1="ClayPaky/gobo00057.svg">Gobo 7</Capability>
   <Capability Min="64" Max="71" Preset="GoboMacro" Res1="ClayPaky/gobo00052.svg">Gobo 8</Capability>
-  <Capability Min="72" Max="113">Fast to slow rotation</Capability>
-  <Capability Min="114" Max="117">Stop rotation</Capability>
-  <Capability Min="118" Max="159">Slow to fast rotation</Capability>
-  <Capability Min="160" Max="255">Gobos shake</Capability>
+  <Capability Min="72" Max="113" Preset="RotationClockwiseFastToSlow">Fast to slow rotation</Capability>
+  <Capability Min="114" Max="117" Preset="RotationStop">Stop rotation</Capability>
+  <Capability Min="118" Max="159" Preset="RotationCounterClockwiseSlowToFast">Slow to fast rotation</Capability>
+  <Capability Min="160" Max="255" Preset="GoboShakeMacro" Res1="">Gobos shake</Capability>
```

Comment:
this is detailed in the manual
160-175 gobo 2
176-191 gobo 3
192-207 gobo 4
208-223 gobo 5
224-239 gobo 6
240-255 gobo 7